webapp/src/app/admin/layout.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="container space-y-4">
+  <div class="flex justify-between gap-8">
+    <div class="flex flex-col gap-4 rounded-md border border-input p-2">
```

Comment:
This box is strangely growing, I think you should just remove `flex flex-col gap-4` and replace it with `space-y-4`. You can also make it sticky so it is always in the top left when scrolling.